src/main/java/nl/mtvehicles/core/movement/PacketHandler.java

```diff
@@ -5,6 +5,7 @@
 import io.netty.channel.ChannelHandlerContext;
 import net.minecraft.server.level.EntityPlayer;
 import net.minecraft.server.network.PlayerConnection;
+import net.minecraft.server.network.ServerCommonPacketListenerImpl;
```

Comment:
Won't this import cause problems in earlier versions?

Reply:
I don't know but I don't want to try it out.

Reply:
:DDD okay